frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -48,6 +48,7 @@ export const Checkbox = forwardRef<CheckboxProps, 'input'>(
         colorScheme={colorScheme}
         ref={ref}
         {...props}
+        aria-label={children?.toString()}
```

Comment:
children is not necessarily a string, it could be a react node. suspect this will not work for the general case

Reply:
oops that was my quick and dirty cheat to see if it would work and I forgot to clean it up HAHAHA let me update

Reply:
[13695af](https://github.com/opengovsg/FormSG/pull/4908/commits/13695aff6b12a2b7385011fd995876c7365c3fc1)